server/project/project.go

```diff
@@ -400,3 +418,19 @@ func (s *Server) GetSyncWindowsState(ctx context.Context, q *project.SyncWindows
 
 	return res, nil
 }
+
+func (s *Server) NormalizeProjs() {
+	ctx := context.Background()
+	projList, err := s.List(ctx, &project.ProjectQuery{})
+	if err != nil {
+		return
+	}
+	for _, proj := range projList.Items {
+		for i := 0; i < 3; i++ {
```

Comment:
We should retry only if update failed due to concurrent modification error. Use `apierr.IsConflict(err)`